packages-web/badge-button/src/BadgeButton.xml

```diff
@@ -1,89 +1,48 @@
-                <enumerationValue key="callMicroflow">Call a microflow</enumerationValue>
-                <enumerationValue key="callNanoflow">Call a nanoflow</enumerationValue>
-            </enumerationValues>
-        </property>
-        <property key="page" type="form" required="false">
-            <caption>Page</caption>
-            <category>Events</category>
-            <description>The page that will be opened on click</description>
-        </property>
-        <property key="openPageAs" type="enumeration" defaultValue="content">
-            <caption>Open page as</caption>
-            <category>Events</category>
-            <description/>
-            <enumerationValues>
-                <enumerationValue key="content">Full page</enumerationValue>
-                <enumerationValue key="popup">Popup</enumerationValue>
-                <enumerationValue key="modal">Blocking popup</enumerationValue>
-            </enumerationValues>
-        </property>
-        <property key="microflow" type="microflow" required="false">
-            <caption>Microflow</caption>
-            <category>Events</category>
-            <description>The microflow that will be executed on click</description>
-            <returnType type="Void" />
-        </property>
-        <property key="nanoflow" type="nanoflow" required="false">
-            <caption>Nanoflow</caption>
-            <category>Events</category>
-            <description>The nanoflow that will be executed on click</description>
-            <returnType type="Void" />
-        </property>
+        <propertyGroup caption="General">
+            <propertyGroup caption="Button">
+                <property key="label" type="textTemplate" required="false">
+                    <caption>Caption</caption>
+                    <description>The caption of the button</description>
+                    <translations>
+                        <translation lang="en_US">Button</translation>
+                        <translation lang="nl_NL">Button</translation>
+                    </translations>
+                </property>
+                <property key="bootstrapStyle" type="enumeration" defaultValue="default">
+                    <caption>Button style</caption>
+                    <description>The appearance of the badge button</description>
+                    <enumerationValues>
+                        <enumerationValue key="default">Default</enumerationValue>
+                        <enumerationValue key="primary">Primary</enumerationValue>
+                        <enumerationValue key="success">Success</enumerationValue>
+                        <enumerationValue key="info">Info</enumerationValue>
+                        <enumerationValue key="warning">Warning</enumerationValue>
+                        <enumerationValue key="danger">Danger</enumerationValue>
+                    </enumerationValues>
+                </property>
```

Comment:
Shouldn't this be a design property?

Reply:
Maybe.

Reply:
We should not make this widget dependent of Atlas because this is not a widget bounded inside a module. As the release is totally separated we should not!

Reply:
Not a design prop till we have support for design props shipped with the widgets